corehq/apps/case_importer/tasks.py

```diff
@@ -24,7 +24,7 @@ def bulk_import_async(config, domain, excel_id):
     try:
         case_upload.check_file()
     except ImporterError as e:
-        update_task_state(bulk_import_async, states.FAILURE, {'errors': get_importer_error_message(e)})
+        update_task_state(bulk_import_async, states.FAILURE, Exception(get_importer_error_message(e)))
```

Comment:
Rather than using `Exception`, which can only be caught with `except Exception` (and that catches almost every other error too), would it make sense to use a more limited error type for these?

I'm assuming this error is raised somewhere, but I can't see it happens in this PR. Maybe my comment doesn't even make sense?

Reply:
The error is actually _not_ ever raised anywhere. Celery just requires that the result for a FAILURE be either an exception or a very specifically formatted dict, and I thought using `Exception` would highlight that it's just filling in for something of type Exception